app/assets/javascripts/darkswarm/services/authentication_service.js.coffee

```diff
@@ -25,9 +25,3 @@ Darkswarm.factory "AuthenticationService", (Navigation, $modal, $location, Redir
     isActive: Navigation.isActive
 
     close: ->
-      if location.pathname in ["/", "/checkout"]
```

Comment:
Initially I just commented out this section to see if it could be removed safely. It adds weird behaviour, like redirecting to the root/home page any time a login modal is closed regardless of what page the user is on, which wasn't helpful for embedded shopfronts where the idea is to stay on the shopfront page. 

Actually I don't really understand the purpose of this code block in any context...?

I can remove the function, yes.